pkg/controller/clone_controller.go

```diff
@@ -16,7 +18,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog"
-	"sigs.k8s.io/sig-storage-lib-external-provisioner/v5/controller"
+	"sigs.k8s.io/sig-storage-lib-external-provisioner/v6/controller"
```

Comment:
hmm.. bit confused.. we already have this import available in sig-s-l-e-provisioner with `v6.0.0` release. https://github.com/kubernetes-sigs/sig-storage-lib-external-provisioner/releases/tag/v6.0.0. So that should be enough ,Isnt it @xing-yang